packages/components/organisms/f-checkout/src/components/Selector.vue

```diff
@@ -55,6 +67,13 @@ export default {
             }
 
             return times;
+        },
+
+        /*
+         * Returns true if ASAP is not available
+         */
+        shouldShowPreOrderWarning () {
+            return !this.availableFulfilment.isAsapAvailable;
```

Comment:
I might not have all the context here but would this still show the warning if you pick a time say 1 hour ahead rather than the next day?

Reply:
It's purely if there's no asap slot available. It comes up if you order these places 
![image](https://user-images.githubusercontent.com/24740015/111749866-505bf400-888a-11eb-8370-4cd214998f77.png)